Sources/Fullscreen/MotorTransactionInsuranceConfirmationView/MotorTransactionInsuranceConfirmationView.swift

```diff
@@ -125,6 +142,8 @@ public class MotorTransactionInsuranceConfirmationView: ShadowScrollView {
         bodyLabel.text = viewModel.bodyText
 
         keyValueGrid.configure(with: viewModel.confirmationDetails, titleStyle: .bodyStrong, valueStyle: .body)
+
+        captionLabel.text = viewModel.caption
```

Comment:
Doesn't it work to just set `captionLabel.isHidden = viewModel.caption == nil` here and always include it in the stack view? 🤔

Reply:
Much better! Totally forgot about that property